src/sdk/strategy/cbd-strategy.ts

```diff
@@ -0,0 +1,126 @@
+import { ethers } from 'ethers';
+
+import { CbdTDecDecrypter } from '../../characters/cbd-universal-bob';
+import { Enrico } from '../../characters/enrico';
+import { ConditionSet } from '../../conditions';
+import { DkgClient, DkgRitual, DkgRitualJSON } from '../../dkg';
+import { base64ToU8Receiver, u8ToBase64Replacer } from '../../utils';
+import { Cohort, CohortJSON } from '../cohort';
+
+type StrategyJSON = {
+  cohort: CohortJSON;
+  conditionSet?: ConditionSet;
+};
+
+type DeployedStrategyJSON = {
```

Comment:
At what point do we think these objects will be deprecated and removed?

Reply:
that's a very good point. Separate them out, finalise the CBD design, remove legacy code 🚀